src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -114,7 +114,10 @@ static ExperimentalFeature()
                     description: "New parameter set for ForEach-Object to run script blocks in parallel"),
```

Comment:
@adityapatwardhan, your last commit had 18 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Set/New/Remove-Service cmdlet tests.New-Service can create a new service called 'testmanual'
```powershell
A parameter cannot be found that matches parameter name 'SecurityDescriptorSddl'.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Set-Service.Tests.ps1: line 232
232:             $service = New-Service @parameters

```
Set/New/Remove-Service cmdlet tests.New-Service can create a new service called 'testsddl'
```powershell
A parameter cannot be found that matches parameter name 'SecurityDescriptorSddl'.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Set-Service.Tests.ps1: line 232
232:             $service = New-Service @parameters

```
Set/New/Remove-Service cmdlet tests.NewServiceCommand can be used as API for 'SecurityDescriptorSddl' with 'D:(A;;CCLCSWRPWPDTLOCRRC;;;SY)(A;;CCDCLCSWRPWPDTLOCRSDRCWDWO;;;BA)(A;;CCLCSWLOCRRC;;;SU)'
```powershell
The property 'SecurityDescriptorSddl' cannot be found on this object. Verify that the property exists and can be set.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Set-Service.Tests.ps1: line 174
174:         $newServiceCommand.$parameter = $value

```
assignment statement parsing.Parse error expected: <<$cricket ?= $soccer>>.Error count
```powershell
Expected 1, but got 2.
at <ScriptBlock>, D:\a\1\s\test\tools\Modules\HelpersLanguage\HelpersLanguage.psm1: line 89
89:         It "Error count" { $errors.Count | Should Be $expectedErrors.Count }
```
assignment statement parsing.Parse error expected: <<$cricket ?= $soccer>>.Error position (iteration:0)
```powershell
Expected 0, but got 10.
at <ScriptBlock>, D:\a\1\s\test\tools\Modules\HelpersLanguage\HelpersLanguage.psm1: line 105
105:             It "Error position (iteration:$i)" -Pending:$SkipAndCheckRuntimeError { $acutalPostion | Should Be $expectedOffsets[$i] }
```